HerPublicWebsite/Views/ReferralRequestFollowUp/ReferralRequestFollowUpAlreadyResponded.cshtml

```diff
@@ -0,0 +1,17 @@
+@using HerPublicWebsite.Controllers
+@{
+    ViewBag.Title = "Response already recorded";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Response already recorded</h1>
+        <p class="govuk-body">
+            You have already submitted a response for this page.
+        </p>
+        <p class="govuk-body">
+            If you think you should not be seeing this page,
+            <a href="@Url.Action(nameof(StaticPagesController.Index), "StaticPages")" class="govuk-link">speak to an advisor</a> for help.
+        </p>
+    </div>
+</div>
```

Comment:
Have we agreed this copy? It reads a little oddly to me, but not sure if we have UX resource to agree this with

Reply:
Not yet, We originally had copy for this to all be sent in an email, when it was realised this wasn't possible I put these pages together with the aim to show to the client and get some feedback on. 

I have screenshots of the pages with their copy on the Jira ticket, which Gareth has taken a look at, but we're still waiting for full confirmation that they are happy with this approach in general.